### DISCUSSION TITLE
Testing Mapper data  after supplier data upload to the Mapper

### DISCUSSION THREAD

**Discussion by hasmikharutyunyan** [happy-travel/docs#14] [General]
After uploading supplier data to the `Mapper` :
(Steps are same for `staging` and `production`)

1. _Mapper team_ must provide changed accommodations list ( accommodation Id, accommodation Name)
2. _QA team_ check data from this list by request to the `osaka` (Insomnia - Predictions/Search For Predictions)
3. _QA team_ check raw data from this list by request to the `nakijin` (Insomnia - StaticDataMapper/Admin/Accommodations/Get Detailed Accommodation)

- If not correct data found  _QA team_ must write about that in `mapper-and-predictions` discord chat.
- If in prediction exists duplicates in which data are all the same _QA team_ must write about that in `mapper-and-predictions` discord chat.

**Comment by oleg-n-k** [comment]
1. Compare the number of accommodations in a connector's DB with the number of accommodations in the RawAccommodations table, Mapper DB.
2. Check the new created and mapped accommodations in the Accommodations table, Mapper DB
If there are no new accommodations in the Accommodations table check the LocalityCandidates table.
The Mapping Process can not be performed for some accommodation if there are no corresponding localities in the Localities  table. The new localities can be moved to the Localities table from the LocalityCandidates table via the relevant endpoint. Also, a locality-candidate can be moved to the locality synonyms of the specific locality.
The mapping process must be restarted after adding new localities or synonyms.
3. Check the result of the mapping by searching the new accommodations by name. First of all, check the accommodations located in priority countries: United Arab Emirates, Saudi Arabia, Maldives, Egypt, Oman, Bahrain.
If the same accommodations exist they have to be sent to the office for merging.
